acct-group/i2p/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>tharvik@gmail.com</email>
+	<name>Valérian Rousset</name>
+</maintainer>
+<maintainer type="project">
+	<email>java@gentoo.org</email>
```

Comment:
Did you have their consent on being added here? You shouldnt add projects just because your program uses, say java or python.

Reply:
I copied it from net-vpn/i2p, should I keep proxy-maint only then?

Reply:
Hmm, if it was there, leave it. Doesnt seem like they are very responsive either though.